contents/handbook/people/team-structure/core-experience.md

```diff
@@ -5,13 +5,53 @@ showTitle: true
 hideAnchor: false
 ---
 
+In PostHog, as a company of [product-minded engineers](/blog/turning-engineers-into-product-people), we believe if you're 
+not having a good experience with our product, we're not doing a good job. No matter if it's because we don't have the 
+features you need, or the existing features do not spark enough joy.
+
+Thus, the multifunctional team **responsible for the overall quality, feature-set and user experience of PostHog's webapp** 
+is not called "Team App", but "Team Core Experience".
+
+Obviously one team can't do all of this alone. PostHog consists of multiple small teams who are specialised in various
+slices of the app, such as [Team Core Analytics](/handbook/people/team-structure/core-analytics) (analytic queries at scale), 
+[Team Growth](/handbook/people/team-structure/growth-engineering) (feature flags, AB testing), and 
+[Team Platform](/handbook/people/team-structure/platform) (plugins, deployments). 
+
+In addition to working on features ourselves, Team Core Experience provides technical alignment to the rest of the 
+organisation in matters of product design, user experience, and frontend and API tooling.
+
 ## People
 
 [See team structure page](/handbook/people/team-structure/team-structure)
 
 ## Mission
 
-To create the smoothest experience when discovering insights about products and its end users.
+Team Core Experience aims to:
+
+- Implement PostHog's vision of increasing the number of successful products in the world, by building features that will move us closer to said vision.
+- Maintain a coherent strategy for frontend and API development (technologies, tools and methodology).
+- Always be in touch with product leadership and our customers, in order to figure out what works, what doesn't, and what needs to be built next.
+- Actually build those features.
+- Support other teams in PostHog in maintaining a coherently high level of product experience acrosse the board.
```

Comment:
```suggestion
- Support other teams in PostHog in maintaining a coherently high level of product experience across the board.
```

Reply:
Suggest we change `formalising` to `formalizing` to adhere to our standard of "american english spellings"